src/main/java/de/adesso/kicker/statistics/trackedranking/persistence/TrackedRanking.java

```diff
@@ -0,0 +1,34 @@
+package de.adesso.kicker.statistics.trackedranking.persistence;
```

Comment:
Maybe trackedRanking and rankig could be in the ame package.
I think an additional statistics package only for this reason is unnecessary.

Reply:
This is mostly to prevent circular dependencies and because they're similar but not really the same. They also don't depend on each other. `TrackedRanking` only uses `Ranking` but not the other way around.

Reply:
So could trackedranking not be a subpackage of ranking?

Reply:
I just noticed something went wrong with the package structure. The `statistics` packages originally hat `ranking` and `trackedranking` in it.